### DISCUSSION TITLE
Azure Cluster Update to 1.25

### DISCUSSION THREAD

**Discussion by Lory1990** [mia-platform/community#244] [Q&A] [answered]
### Which section?

Console

### Which is the version?

latest

### Description

Hi!
I want to update my kubernetes cluster to 1.25.6 but azure is showing warning about this 

![image](https://user-images.githubusercontent.com/7715601/236208853-0480edd9-577e-4acf-9cf9-71d47e0d36aa.png)

Do mia platform uses one of this Api?

I am sure about CronJob - I already updated to batch/v1

**Comment by davidebianchi** [comment, marked as answer]
Hi @Lory1990! 
Mia-Platfom Console uses only the CronJob and HPA APIs. But if you have already updated the CronJob it should works (since we use HPA at version 1, not the v2 at the moment).
The support of K8s 1.25 will be officially added with the Mia-Platform v11, which we will release on May 11th (we will create and manage CronJob at v1).

**Reply by Lory1990** [reply]
Thanks @davidebianchi  we will wait for mia platform 11
